src/routes/e2e/widget/modal/+page.svelte

```diff
@@ -55,78 +59,165 @@
       const response = await fetch(`${window.location.origin}/api/locale`);
       content = response.ok && (await response.json());
     }
-
-    config.set({
-      forgerock: {
-        clientId: 'WebOAuthClient',
-        redirectUri: `${window.location.origin}/callback`,
-        scope: 'openid profile email me.read',
-        serverConfig: {
-          baseUrl: 'https://openam-sdks.forgeblocks.com/am/',
-          timeout: 5000,
+    if (journeyParam?.includes('PingProtect')) {
```

Comment:
I'm having a hard to distinguishing the difference between the logic run when truthy and when falsey. Can you point me to the difference?

Reply:
The AM Url.

Reply:
Oh, okay, could we just assign the URL to a variable and conditionally set a value on it, rather than wrapping to whole config in the condition? It would reduce all the duplication and noise in this file.